qcodes/instrument_drivers/AlazarTech/ATS9440.py

```diff
@@ -119,7 +129,7 @@ def __init__(self, name, **kwargs):
                                         'TRIG_ENGINE_OP_J_XOR_K': 4,
                                         'TRIG_ENGINE_OP_J_AND_NOT_K': 5,
                                         'TRIG_ENGINE_OP_NOT_J_AND_K': 6})
-        for i in ['1', '2']:
+        for i in ['1', '2','3','4']:
```

Comment:
Trigger is only 2 iterations, exactly like ATS9360. No need to change it to 4.  Only the loop for channels needs to be changed to 4 iterations as we have 4 channels (you have correctly changed it above). Please revert this change back.

Reply:
Yea my mistake. Wasn't supposed to be changed. Was just curious what the errors would be.